cfme/utils/appliance/__init__.py

```diff
@@ -179,21 +179,19 @@ def configure_appliance_dedicated_db(self, region, username, password, dbname, d
             " --dbname {dbname} --verbose --dbdisk {dbdisk} --key --standalone".format(
                 region=region, username=username, password=password, dbname=dbname, dbdisk=dbdisk))
 
-    def configure_ipa(self, ipaserver, username, password, domain, realm):
-        self._run("--ipaserver {ipaserver} --ipaprincipal {username} --ipapassword {password}"
-            " --ipadomain {domain} --iparealm {realm}".format(
-                ipaserver=ipaserver, username=username, password=password, domain=domain,
-                realm=realm))
-        assert self.appliance.ssh_client.run_command("systemctl status sssd | grep running")
-        return_code, output = self.appliance.ssh_client.run_command(
-            "cat /etc/ipa/default.conf | grep 'enable_ra = True'")
-        assert return_code == 0
+    def configure_ipa(self, ipaserver, username, password, domain=None, realm=None):
```

Comment:
Mike wouldn't it be better to move all auth related stuff to appliance plugin ?

Reply:
Really like this idea, discussed offline, **optional** for this PR - but we'll be moving things later.